### DISCUSSION TITLE
WPF CMYK with ICC profiles

### DISCUSSION THREAD

**Discussion by CustomBrandCo** [ElinamLLC/SharpVectors#260] [Ideas]
**Can Sharp Vectors currently handle CMYK SVGs and the resulting WPF have the CMYK value?**

SVGs are able to use ICC colour profiles to specify CMYK colours as shown here:

`<color-profile name="acmecmyk" xlink:href="http://printers.example.com/acmecorp/model1234"/>`
    `<circle fill="#CD853F icc-color(acmecmyk, 0.11, 0.48, 0.83, 0.00)"/>`
[https://www.w3.org/TR/SVGColor12/](url)

And I know that WPF can handle CMYK as shown in this example:

`<Grid Background="ContextColor file://C:/CoatedFOGRA39.icc 1.0,0.0,0.0,1.0,1.0">`

`var profilePath = "C:\\CoatedFOGRA39.icc";`
`var c = Color.FromValues(new [] {1,0,0,0}, new Uri(profilePath, UriKind.Absolute));`

**Comment by paulushub** [comment]
@CustomBrandCo Thanks for reporting it. ICC profiles are supported in static images but not in colors.
Can you take it up and provide a PR?
